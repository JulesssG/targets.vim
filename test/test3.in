A
a ( b ) c

B ( a ) b
a ( b ) C

D ( a
b ) c

a ( b
c ) E

a ( b
F ( ) c
d ) e

a ( b
c ( ) G
d ) e

a ( b ) c
H " this must be the last pair test

