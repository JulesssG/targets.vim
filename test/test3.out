A
a (A) c

B (B) b
a (C) C

D (D) c

a (E) E

a (F) e

a (G) e

a ( b ) H (H) d

a (I) c
I " this must be the last pair test

